crates/ibc/src/core/ics04_channel/handler/timeout.rs

```diff
@@ -119,26 +121,33 @@ where
             }
             let seq_recv_path_on_b =
                 SeqRecvPath::new(&msg.packet.port_id_on_b, &msg.packet.chan_id_on_b);
-            client_state_of_b_on_a.verify_next_sequence_recv(
-                msg.proof_height_on_b,
+
```

Comment:
on line 82, we should be checking if the client is frozen

Reply:
[828b40a](https://github.com/cosmos/ibc-rs/pull/531/commits/828b40a5a04d351041b190841fd281e9ac68f467)